membership_extension/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Membership extension
+====================
+
+* Membership Category
```

Comment:
Include a preface saying: `This module extends Odoo's membership management with the following features:`